demo/node-javascript/src/index.js

```diff
@@ -89,6 +89,49 @@ async function main() {
       console.log(`  URL: ${fileInfo.url}`);
     }
   }
+
+  // Example: Update file metadata
+  console.log('\n=== File Update Operations ===\n');
```

Comment:
Just a nitpick, but could we replace all of these with something like:

```
  console.log(os.EOL);
  console.log('=== File Update Operations ===');
  console.log(os.EOL);
```

So that it's cleaner to read and works on windows?